glide.yaml

```diff
@@ -107,3 +107,7 @@ import:
   version: v0.9.0
 - package: github.com/gorhill/cronexpr
   version: f0984319b44273e83de132089ae42b1810f4933b
+- package: github.com/pierrec/lz4
```

Comment:
I assume these 2 new dependencies come from sarama 1.11? They have tons of binaries inside. Should we enhance our filter to filter out some?